protected-root/README.md

```diff
@@ -1,141 +0,0 @@
-[![Build Status](https://travis-ci.org/broadinstitute/gatk-protected.svg?branch=master)](https://travis-ci.org/broadinstitute/gatk-protected)
-[![Coverage Status](https://coveralls.io/repos/broadinstitute/gatk-protected/badge.svg?branch=master&service=github)](https://coveralls.io/github/broadinstitute/gatk-protected?branch=master)
-[![License (3-Clause BSD)](https://img.shields.io/badge/license-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
-
-GATK4-Protected (codename Hellbender-protected)
-===============================================
-
-GATK4 development of the previously license-protected part of the toolkit. The contents of this repo will be merged into broadinstitute/gatk in the near future.
-
-This README is aimed at developers.  For user information, please see the [GATK 4 forum](http://gatkforums.broadinstitute.org/gatk/categories/gatk-4-alpha)
-
-Read GATK 4 README
-------------------------
-
-**Please refer to the [GATK 4 public repo README](https://github.com/broadinstitute/hellbender/blob/master/README.md) for general guidelines and how to setup your development environment.**
-
-Requirements
-------------
-* R 3.1.3 see additional requirements below: [R package requirements](#r-required-packages)
-
-* Java 8
-
-* (Developers) Gradle 2.13 is needed for building the GATK. We recommend using the `./gradlew` script which will
-download and use an appropriate gradle version automatically.
-
-* (Developers) git lfs 1.1.0 (or greater) is needed for testing GATK-Protected builds.  It is needed to download large files for the complete test suite. Run ``git lfs install`` after downloading, followed by ``git lfs pull`` to download the large files. The download is ~500 MB.
```

Comment:
Please add that `git lfs pull` must be run from the repository root as mentioned in https://github.com/broadinstitute/gatk/issues/2978.